### DISCUSSION TITLE
Generating images without UI

### DISCUSSION THREAD

**Discussion by FelixDeMan** [lllyasviel/Fooocus#2731] [Q&A] [answered]
Hi!

I love what you did here! For our use-case, we're trying to run the script for inpaint on masks that we automatically generate, but we are facing quite some issues to implement this, my guess is that most of these are related to the inpaint patch function. 

Did anyone manage to make it work?

**Comment by mashb1t** [comment, marked as answer]
Please check out the API in https://github.com/lllyasviel/Fooocus/discussions/2613#discussioncomment-8973861 or find more information in discussions/issues/PRs for "api".
In general, the Gradio API is the only way to use Fooocus without the browser interface. You can also find more detail about API calls in the footer.

**Comment by FelixDeMan** [comment]
Hi, thanks for your reply! After some more searches I got to this repo: https://github.com/mrhan1993/Fooocus-API.


It has been working wonders!
